pkg/controller.v1/control/controller_ref_manager.go

```diff
@@ -0,0 +1,377 @@
+// Copyright 2019 The Kubeflow Authors
```

Comment:
If we think it's fine to sync pod_ref_manager, it makes sense to rename to `controller_ref_manager.go`.